mantis-server/mantis-server-worker/src/main/java/io/mantisrx/server/worker/jobmaster/JobMasterService.java

```diff
@@ -105,8 +104,6 @@ private Measurements handleMetricEvent(final String ev) {
 
         } catch (JsonProcessingException e) {
             logger.error("failed to parse json", e);
-        } catch (IOException e) {
```

Comment:
Is this a spurious change?

Reply:
it was a warning during gradle build, will remove and move to a follow-up PR.